examples/angular/yarn.lock

```diff
@@ -4298,6 +4298,14 @@ karma-jasmine@2.0.1:
   dependencies:
     jasmine-core "^3.3"
 
+karma-junit-reporter@2.0.1:
```

Comment:
We need at least one example that doesn't include this dependency to verify that we didn't accidentally introduce a breaking change.

My preference is to just update one of the examples/ to include the dependency so it's possible to observe the XML output from a karma test.

Reply:
I've removed the dependency from angular_view_engine and protocol_buffers and left it in web_testing and angular (since this is the one I'm primarily interested in).

Reply:
cool as long as there's a mix